### DISCUSSION TITLE
TCP congestion window

### DISCUSSION THREAD

**Discussion by YafSQ1** [iovisor/bcc#4621] [Ideas]
Hello,

i found the tcpcong.py and its awesome!

I am writing to make a request. It would be fantastic if you could include the current CWND value (Congestion Window) into tcpcong.py or into another script. The CWND value is a crucial parameter in the Congestion Control mechanism and significantly impacts the performance of a TCP connection. Is it possible to add the snd_cwnd,snd_ssthresh to it?

Thanks.

**Comment by chenhengqi** [comment]
This should be easy to implement, could you help with a PR ?

**Reply by YafSQ1** [reply]
I would like to, but I am not very good at programming. Tried it to do it myself, but failed. Because of that I made the request

**Reply by chenhengqi** [reply]
Please raise an issue instead, so that someone in the community might help.
